### PR TITLE
[uikit] Update for Xcode 9 GM

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

File: src/uikit.cs [hunk 1]
```diff
@@ -6106,12 +6106,18 @@ interface UIGestureRecognizerDelegate {
...
 	{
+		[Deprecated (PlatformName.TvOS, 11, 0, message: "Use 'PreferredFormat' instead.")]
 		[Static]
 		[Export ("defaultFormat")]
 		UIGraphicsRendererFormat DefaultFormat { get; }
 	
 		[Export ("bounds")]
 		CGRect Bounds { get; }
+
+		[Watch (4,0), TV (11,0), iOS (11,0)]
```
Comment (spouliot) [inline src/uikit.cs]: is that type included in watchOS ?

Comment (spouliot) [reply src/uikit.cs]: not it's not in watchOS headers, let's remove it to avoid confusion

Comment (VincentDondain) [reply src/uikit.cs]: Yea and actually all that code is under a `#if !WATCH` I didn't see...

**Comment by rolfbjarne** [comment]
Test failures are unrelated (xammac password failure - bug #[58394](https://bugzilla.xamarin.com/show_bug.cgi?id=58394)).

**Pull request merged**